api/settings.js

```diff
@@ -226,6 +227,31 @@ function getSettings(account, callback) {
   });
 }
 
+function setTransactionParameters(transaction, account, settings) {
+  transaction.accountSet(account);
```

Comment:
Isn't this object the `{ tx_json: txJson }` that's passed in by `utils.createTxJSON`